contact.html

```diff
@@ -0,0 +1,75 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Sprint Challenge - About</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+  <link rel="stylesheet" href="css/index.css">
+
+</head>
+
+<body>
+    <div class="container about-page">
+     
+        <header>
+            <nav class="navTop"><img src="img/lambda-black.png" alt="Lambda is the coolest">
+              <a href="index.html">Home</a> 
+              <a href="about.html">About</a> 
+              <a href="products.html">Products</a> 
+              <a href="#">Blog</a> 
+              <a href="contact.html">Contact</a> 
+            </nav>
+      </header>
+
+      <section>
+          <div>
+              <img class="jumbo" src="img/header_hotels_1200x300-800x300.jpg" alt="Image of a code snippet.">
+          </div> 
+        </section>
+
+        <section class="about">
+          <h2>
+              Reach out to us!
+          </h2>
+            Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.        
+        </section>
+    
+        <form>
+            First name:<br><br>
+            <input type="text" name="firstname"><br><br>
+            Last name:<br><br>
+            <input type="text" name="lastname"><br><br>
+            eMail:<br><br>
+            <input type="text" name="firstname"><br><br>
+            Phone:<br><br>
+            <input type="text" name="lastname"><br><br>
+            Address:<br><br>
+            <input type="text" name="firstname"><br><br>
+            Note:<br><br>
+            <input class="note" type="text" name="lastname">
+          </form> 
+    
```

Comment:
You can style these text inputs to make them a bit more visually appealing. Maybe give them some more width, background color, padding, font-size, font color.

Reply:
Here's a good tutorial on styling forms:
https://www.w3schools.com/css/css_form.asp